doc/source/news.rst

```diff
@@ -3,6 +3,34 @@
 News
 ====
 
+.. _release-12-07:
+
+Release 12.07 - 2022-08-30
+--------------------------
+
+Improvements
+^^^^^^^^^^^^
+
+* [:doc:`/install/centos`][:doc:`/install/almalinux`] Added support for MariaDB 10.3.36.
+
+Fixes
+^^^^^
+
+* Fixed a bug to fail to install or upgrade Mroonga.[GitHub #525][Reported by Tomohiro KATO][Patched by Jérome Perrin]
```

Comment:
This bug occurs when we install or upgrade Mroonga with `install.sql`.
This bug doesn't occur when we install or upgrade Mroonga with a package management system(e.g. apt, yum). 
Because `install_plugin.sql` and `update.sql` are used when we install or upgrade Mroonga with a package management system.

Reply:
I see.

Reply:
I have fixed it.